services/wfm/src/main/java/org/openkilda/wfm/topology/islstats/IslStatsTopology.java

```diff
@@ -53,14 +52,16 @@ public StormTopology createTopology() {
                 .shuffleGrouping(ISL_STATS_SPOUT_ID);
 
         String openTsdbTopic = topologyConfig.getKafkaOtsdbTopic();
-        checkAndCreateTopic(openTsdbTopic);
         KafkaBolt openTsdbBolt = createKafkaBolt(openTsdbTopic);
         builder.setBolt(ISL_STATS_OTSDB_BOLT_ID, openTsdbBolt, topologyConfig.getParallelism())
                 .shuffleGrouping(ISL_STATS_BOLT_ID);
 
         return builder.createTopology();
     }
 
+    /**
+     * main.
```

Comment:
Is that workaround for checkstyle?

Reply:
I'd recommend the following text: Entry point for local run of the topology.